src/ServiceControl/Infrastructure/WebApi/RootController.cs

```diff
@@ -39,7 +39,8 @@ public OkNegotiatedContentResult<RootUrls> Urls()
                 LicenseStatus = license.IsValid ? "valid" : "invalid",
                 LicenseDetails = baseUrl + "license",
                 Configuration = baseUrl + "configuration",
-                EventLogItems = baseUrl + "eventlogitems"
+                EventLogItems = baseUrl + "eventlogitems",
+                ArchivedGroupsUrl = baseUrl + "errors/groups/{classifier?}"
```

Comment:
Are we deliberately not adding `archive/groups/id/{groupId}` here or did I miss that?

Reply:
Nope, added